### PR TITLE
winPB: Update ccert role to get windows.p12 from vendor_files

### PR DESCRIPTION
**Pull request by Willsparker**
Fixes: #1558 
ref: #1556 
In draft until #1560 is merged.

Retrieve the `windows.p12` file from the vendor files that are retrieved from the `Get_Vendor_Files` role that is put in with #1560 . Update the place in which the `windows.p12` file is located, to `C:/openjdk`

### CODE REVIEW

**Comment by karianna** [comment]
@Willsparker not sure if that ansible lint check failure is really related but probably needs looking at regardless

**Comment by Willsparker** [comment]
@karianna Can't say I've seen the error before- I'll run `ansible-lint` locally just to confirm :-)

**Review by gdams — APPROVED** [review]

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/CodesignCert/tasks/main.yml [hunk 1]
```diff
@@ -5,18 +5,17 @@
...
-    checksum: 836c302e944cae970b1fa57b9e5cff552e3d61f0ce4445e79185900c1098e40e
-    checksum_algorithm: sha256
+# The vendor files are retrieved by the Get_Vendor_Files role
+- name: Get windows.p12 and put in C:\openjdk Directory
+  win_copy:
+    src: vendor_files/windows.p12
+    dest: C:\openjdk\windows.p12
   when: (not cert_installed.stat.exists)
   tags:
     - ccert
```
Comment (gdams) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/CodesignCert/tasks/main.yml:21]: you should probably add the adoptopenjdk tag to this

Comment (Willsparker) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/CodesignCert/tasks/main.yml:22]: Fixed

**Comment by gdams** [comment]
@Willsparker good to go?

**Comment by Willsparker** [comment]
Yup :-)

**Pull request merged**